File: WebAPI/ClientApp/src/app/models/AvailabilityRequestModelForCustomer.ts
import { ImageModel } from "./ImageModel";

export class AvailabilityRequestModelForCustomer {
    id: number;
    advertName: string;
    created: Date;
    updated: Date;
    address: string;
    conditions: string;
    sum: number;
    landlordName: string;
    phone: string;
    requestStateId: number;
    userId: number;
    images: ImageModel[];
    startRent: Date;
    endRent: Date;

    public constructor(_id: number, _advertName: string, _created: Date, _updated: Date, _address: string, _conditions: string, _sum: number, 
        _landlordName: string, _phone: string, _requestStateId: number, _userId: number, _images: ImageModel[], _startRent: Date, _endRent: Date) {
        this.id = _id;
        this.advertName = _advertName;
        this.created = _created;
        this.updated = _updated;
        this.address = _address;
        this.conditions = _conditions;
        this.sum = _sum;
        this.landlordName = _landlordName;
        this.phone = _phone;
        this.requestStateId = _requestStateId;
        this.userId = _userId;
        this.images = _images;
        this.startRent = _startRent;
        this.endRent = _endRent;
    }
}

File: WebAPI/Models/AvailabilityRequestModelForCustomer.cs
using System;
using System.Collections.Generic;

namespace WebAPI.Models
{
    public class AvailabilityRequestModelForCustomer
    {
        public int Id { get; set; }
        public string AdvertName { get; set; }
        public DateTime Created { get; set; }
        public DateTime Updated { get; set; }
        public string Address { get; set; }
        public string Conditions { get; set; }
        public int Sum { get; set; }
        public string LandlordName { get; set; }
        public string Phone { get; set; }
        public int RequestStateId { get; set; }
        public int UserId { get; set; }
        public List<ImageModel> Images { get; set; }
        public DateTime StartRent { get; set; }
        public DateTime EndRent { get; set; }
    }
}
